src/proc/motion-transform.cpp

```diff
@@ -50,14 +50,23 @@ namespace librealsense
         if (mm_calib)
         {
             _imu2depth_cs_alignment_matrix = (*mm_calib).imu_to_depth_alignment();
+
             if (_mm_correct_opt)
             {
-                auto accel_intr = (*mm_calib).get_intrinsic(RS2_STREAM_ACCEL);
-                auto gyro_intr = (*mm_calib).get_intrinsic(RS2_STREAM_GYRO);
-                _accel_sensitivity  = accel_intr.sensitivity;
-                _accel_bias         = accel_intr.bias;
-                _gyro_sensitivity   = gyro_intr.sensitivity;
-                _gyro_bias          = gyro_intr.bias;
+                if (mm_calib->is_intrinsic_valid())
```

Comment:
It will call HW continuously if the option is not null but the actual table is not valid. 
It should be refactored : if there is no intrinsic then the mm_corect_opt is disabled

Reply:
removed